### DISCUSSION TITLE
2022/10/24/railsbook-v2-ruby-ecosystem

### DISCUSSION THREAD

**Discussion by giscus** [kaochenlong/kaochenlong-discuss#4] [General]
# 2022/10/24/railsbook-v2-ruby-ecosystem

第一版的「為你自己學 Ruby on Rails」是 2017 年的事，轉眼已過了五年，Ruby 的主流版本從原本的 Ruby 2 系列變成 Ruby 3 系列，而網站開發框架 Ruby on Rails（本書將會以 Rails 簡稱）也從版本 5 昇級到版本 7，除了 Rails 本身的功能有不小的進步外，Rails

https://kaochenlong.com/2022/10/24/railsbook-v2-ruby-ecosystem.html

**Comment by moksha-z** [comment]
写的蛮好的嘞，这个系列博主是没有再更新了吗？

**Reply by kaochenlong** [reply]
別擔心，Ruby 目前還是我的最愛，只是最近被別的事情卡住了，把手邊任務清掉之後就會再回來更新了 :)
